src/database/operations/baseFieldLocalization/loadBaseFieldLocalizationsBundle.ts

```diff
@@ -5,21 +5,20 @@ import type {
 	BaseFieldLocalization,
 } from '../../../types';
 
-export const loadBaseFieldLocalizationsBundle = async (queryParameters: {
-	offset?: number;
-	limit?: number;
-	baseFieldId?: number;
-}): Promise<Bundle<BaseFieldLocalization>> => {
-	const defaultQueryParameters = {
-		baseFieldId: 0,
-		offset: 0,
-		limit: 0,
-	};
+export const loadBaseFieldLocalizationsBundle = async (
+	offset: number,
+	limit: number,
+	options: {
+		baseFieldId?: number;
+	} = {},
```

Comment:
Is `baseFieldId` ever missing in the places we call it? Do we need to maintain that flexibility? If not, we can make `baseFieldId` a required positional parameter, too, like `limit` and `offset`. And, since apparently I have opinions on parameter ordering, having it be first seems the most readable to me: `loadBaseFieldLocalizationsBundle(baseFieldId, limit, offset)`.

Reply:
I think in this case we did resolve that we'll never want to allow a person to load ALL base field localizations at once so you're absolutely right!

Also, I can't respond to the top level comment directly but will say I agree with the philosophy: if something can be required it doesn't go in the `options`.

Reply:
I actually don't think the suggest change applies to all of the loaders though -- in the vast majority of cases the option is optional at the query param level (e.g. you don't HAVE to specify `fooId` or `whatever the filter is` so while a value is being passed, that value is generally `string | undefined` or `number | undefined` so in most cases we'll keep the options.)

Reply:
I am somewhat tempted, in this case, to add `ByBaseFieldId` to the operation name to signal that there is no way to generate a broader bundle

Reply:
Update: we talked through this and the bottom line is....

being able to receive undefined does not an optional parameter make!

Which is to say: by the time we get to the operation, we know the shape of the data.  Just because something can be `undefined` doesn't mean it should be an optional parameter (it's just that `undefined` can be an expected value for that parameter, which may have meaning internally).

All that is to say, I'll take out the concept of `options` on all of the handlers and instead we'll use named parameters.